apps/fluent-tester/src/FluentTester/theme/ThemePickers.tsx

```diff
@@ -18,7 +18,7 @@ export const themePickerStyles = StyleSheet.create({
 
   dropdown: {
     height: 30,
-    width: 90,
+    width: 120,
```

Comment:
This would affect all platforms, so you might want to make this android specific. 
On iOS I personally want to get rid of all the dropdowns and hide them behind a modal, because they look really bad on that platform.

Reply:
will pick up this work item after this  [PR#633](https://github.com/microsoft/fluentui-react-native/pull/633) is merged